ruleset/rules/0995-maltiverse_rules.xml

```diff
@@ -0,0 +1,8 @@
+<group name="maltiverse,">
+    <rule id="99500" level="0">
```

Comment:
```suggestion
    <rule id="99500" level="10">
```
We should define a base level, e.g. 10, for the rule to trigger alerts when Maltiverse returns aggregated information.